addons/im_livechat/static/src/models/discuss/discuss.js

```diff
@@ -25,4 +25,11 @@ addFields('mail.discuss', {
         inverse: 'discussAsLivechat',
         isCausal: true,
     }),
+    /**
+     * All livechats that is known at present.
```

Comment:
is -> are (and "at present" is a bit redundant, it's no more or less present than any other field)